### PR TITLE
Set directories/files to read only before deleting

### PR DESCRIPTION
**Pull request by Haacked**
This is inspired by a bug found in GitHub Desktop for Windows.

### CODE REVIEW

File: src/DirectoryInfo.cs [hunk 1]
```diff
@@ -75,6 +76,41 @@ public IDirectoryInfo CreateSubdirectory(string path, System.Security.AccessCont
...
+                }
+
+                foreach (var dir in dirs)
+                {
+                    dir.Delete(true);
+                }
+            }
+
+            // Make sure the directory is not readonly
+            inner.Attributes = FileAttributes.ReadOnly;
```
Comment (shiftkey) [inline src/DirectoryInfo.cs]: This should be `FileAttributes.Normal`

**Comment by shiftkey** [comment]
:thumbsup: just that minor nit

File: src/DirectoryInfo.cs (hunk 1 above)
Comment (Haacked) [inline src/DirectoryInfo.cs]: LOLOLOLOL!

**Pull request merged**